unused_deps/unused_deps.go

```diff
@@ -116,9 +117,54 @@ func inputFileName(blazeBin, pkg, ruleName, extension string) string {
 	return fmt.Sprintf("%s/%s/%s.%s", blazeBin, pkg, ruleName, extension) // *_{binary,test}
 }
 
-// directDepParams returns --direct_dependency entries from paramsFileName (a jar-2.params file)
-// as a map from jar files to labels.
-func directDepParams(paramsFileNames ...string) (depsByJar map[string]string) {
+// targetLabel reads the manifest of the jar specified by jarFileName.  If the
+// manifest has a "Target-Label" property, targetLabel returns its value.
+func targetLabel(jarFileName string) (string, error) {
+	const BAZEL_TARGET_KEY = "Target-Label:"
```

Comment:
up to you: make this a file-level const

style: still use lowerCamelCase for variables, even constants